src/tests/testinput/3dvar_sic.yaml

```diff
@@ -7,8 +7,49 @@ cost function:
   background error:
     covariance model: SABER
     saber central block:
-      saber block name: ID
-  geometry :
+#      saber block name: ID
```

Comment:
I think I am misunderstanding how this all fits together, but I was wondering if the changes in here are now covering additional code to before, whereas before they were covering a smaller part of the interface? If so, I would favour adding an extra test that tests against saber alongside the one we had before. As long as the test is running in around the 10 second mark, I think it can be handy for debugging to have coverage of smaller blocks of code in the test where possible. i.e where have I missed something in the interface to OOPS vs missing something in the interface to SABER?

The above might not make sense though, so I am very open to your views on it!

Reply:
I anticipated this comment and had already added the original test back but now called 3dvar_sic_identity. Because I changed the old test and then created a new test with the old settings this wasn't clear from the diffs.